test/dlc_tests/sanity/test_torchvision.py

```diff
@@ -10,7 +10,7 @@
 
 
 @pytest.mark.model("N/A")
-def test_torchvision_nms_training(pytorch_training):
+def test_torchvision_nms_training(pytorch_training, pt16_and_above_only):
```

Comment:
This test applies to all PyTorch versions.

It is meant to test that the custom-torchvision binary has been correctly installed in the DLC.

Reply:
nms does not seems to be in torchvision for v1.5.0
we get such error `RuntimeError: No such operator torchvision::nms`

Reply:
Pytorch team will be built the `torchvision` binary again to solve this issue. Will revert the test skip.

Reply:
updated the torchvision binary.